internal/cloud/realm/app_test.go

```diff
@@ -50,3 +51,16 @@ func TestRealmApps(t *testing.T) {
 		})
 	})
 }
+
+func setupTestApp(t *testing.T, client realm.Client, groupID, name string) (realm.App, func()) {
+	t.Helper()
+	app, err := client.CreateApp(groupID, name, realm.AppMeta{})
+	assert.Nil(t, err)
+	teardown := func() {
+		deleteErr := client.DeleteApp(groupID, app.ID)
+		if deleteErr != nil {
+			t.Log(fmt.Sprintf("failed to delete test app with groupID: %v and ID: %v", groupID, app.ID))
```

Comment:
should be able to just use `t.Logf` here... lets rework this message too:

```go
t.Logf("warning: failed to delete test app (id: %s): %s", app.ID, err)
```

Reply:
Perfect, learned something new today!